src/pages/active.tsx

```diff
@@ -0,0 +1,64 @@
+import { FC, useState, useEffect } from 'react';
+import Head from '@/components/head';
+import Layout from '@/components/Layout';
+import Card from '@/components/tasks/card';
+import useFetch from '@/hooks/useFetch';
+import classNames from '@/styles/tasks.module.scss';
+import { task } from '@/components/constants/types';
+import Accordion from '@/components/Accordion';
+
+const TASKS_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/tasks`;
+
+const renderCardList = (tasks: task[]) => tasks.map(
+  (item: task) => <Card content={item} key={item.id} />,
+);
+
+const Active: FC = () => {
+  const [tasks, setTasks] = useState<task[]>([]);
+  const [activeTasks, setActiveTasks] = useState<any>(null);
+
+  const {
+    response,
+    error,
+    isLoading,
+  } = useFetch(TASKS_URL);
+
+  useEffect(() => {
+    if ('tasks' in response) {
+      setTasks(response.tasks);
+      const active = tasks.filter(
+        (item: task) => item.status === 'Active',
```

Comment:
We are receiving this from the backend right? Shouldn't the value of status be in lowercase? 
![image](https://user-images.githubusercontent.com/66248039/113037269-d6ddd300-91b2-11eb-9ce8-0de6dd68980d.png)